util/src/main/scala/com/scalableminds/util/geometry/BoundingBox.scala

```diff
@@ -55,6 +55,8 @@ case class BoundingBox(topLeft: Vec3Int, width: Int, height: Int, depth: Int) {
   def *(that: Vec3Int): BoundingBox =
     BoundingBox(topLeft * that, width * that.x, height * that.y, depth * that.z)
 
+  def /(that: Vec3Int): BoundingBox = BoundingBox(topLeft / that, width / that.x, height / that.y, depth / that.z)
```

Comment:
will this floor? should it maybe ceil to avoid that voxels get lost at the border?